services/core-web/common/constants/API.js

```diff
@@ -48,6 +48,7 @@ export const PERMITAMENDMENT = (mineGuid, permitGuid, permitAmendmentGuid) =>
   `/mines/${mineGuid}/permits/${permitGuid}/amendments/${permitAmendmentGuid}`;
 export const PERMITAMENDMENTDOCUMENT = (mineGuid, permitGuid, permitAmendmentGuid, documentGuid) =>
   `/mines/${mineGuid}/permits/${permitGuid}/amendments/${permitAmendmentGuid}/documents/${documentGuid}`;
+export const PERMITDELETE = (mineGuid, permitGuid) => `/mines/${mineGuid}/permits/${permitGuid}`;
 
```

Comment:
should be PERMIT_DELETE... `PERMITAMENDMENTDOCUMENT` is in the incorrect casing, but you dont have to fix that here.